이충녕/정렬/k번째수.js

```diff
@@ -0,0 +1,24 @@
+// 채점 결과
+// 정확성: 100.0
+// 합계: 100.0 / 100.0
+
+// 풀이
+// 1. 2차원 배열의 자식 배열에서 i, j, k 반환
+// 2. 인덱스가 아닌 순서로 접근하기 위해 i - 1 ~ j 범위 컷팅
+// 3. 2번에서 얻은 배열 오름차순 정렬 후 k번째 수 추가
+
+function solution(array, commands) {
+  const kLocation = [];
+
+  commands.forEach((command) => {
+    const i = command[0];
+    const j = command[1];
+    const k = command[2];
```

Comment:
아래처럼 구조분해할당 할 수 있을 것 같아요. 
+ 그리고 i, j, k를 start, end, index와 같이 의미를 담는 건 어떨까요? (코딩테스트는 협업하는 게 아니니 의미를 담지 않아도 된다는 의견도 있지만, 저는 가독성 측면에서 의미가 있으면 더 좋을 것 같아 제안해봅니다!)
```suggestion
  commands.forEach(([i, j, k) => {
```